cmd/hecklerd/main.go

```diff
@@ -1117,12 +1126,21 @@ func groupResources(commitLogId git.Oid, targetDeltaResource *deltaResource, nod
 
 	for nodeName, node := range nodes {
 		if nodeDeltaResource, ok := node.commitDeltaResources[commitLogId][targetDeltaResource.Title]; ok {
-			// TODO: cmp is not recommended to be used in production, because it
-			// panics on any errors. It would probably be better to write a custom
-			// compare function which can account for skipping fields which are not
-			// relavant to the desired measure of equality, e.g. source file line
-			// number
-			if cmp.Equal(targetDeltaResource, nodeDeltaResource, cmpopts.IgnoreFields(deltaResource{}, "Line")) {
+			// We want to compare everything except the Line field. We can take
```

Comment:
Should we functionalize this?  Is there a reason not to?

Reply:
If you mean just the code I added, then it feels like something that isn't generic and wouldn't get reused, so I didn't feel the need to make it be its own bespoke function. I feel like I'm missing something here, though–do you mean a larger chunk of code than what I changed?